### ISSUE TITLE
Uploading Accessibility Issues

### ISSUE THREAD

**Issue by lai-kevin** [MAP-Dot-X/Accessibility-Issue-Reporter-Timemap#2]
Current Behavior
----------------

Currently a user cannot upload an accessibility issue. There is also no user management system.

Expected Behavior
-----------------

A user can sign-in/sign-up by clicking on a button on the left toolbar. Once signed-in, the user can click on a "Report Issue" button on the left toolbar which presents the user with a modal containing a form and the ability to upload/take a picture. 

Another ticket has been made for integrating Clerk into Timemap for user management. Consult with the contributor of that ticket to add support of users.

The form should ask for the following information:
- Location
- Tags
- Severity
- Description
- Title
- Picture
